addons/l10n_sa_edi/views/account_move_views.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="view_move_form" model="ir.ui.view">
+        <field name="name">account.move.l10n_sa_edi.form</field>
+        <field name="model">account.move</field>
+        <field name="inherit_id" ref="account.view_move_form"/>
+        <field name="arch" type="xml">
+            <button name="button_cancel_posted_moves" position="attributes">
+                <attribute name="invisible">1</attribute>
```

Comment:
Hmm, this hack should be refined a bit as it will do weird things for non-SA companies and we might simply inherit edi_show_cancel_button's compute method

Reply:
@jco-odoo The invisible="1" doesn't really make sense as we already override the _compute_edi_show_cancel_button method, so this was removed